docs/source/autoscaling.rst

```diff
@@ -101,6 +118,37 @@ The currently available decicion policies are:
   An external process should periodically decide how many replicas this service needs to run, and use the Paasta API to tell Paasta to scale.
   See the :ref:`How to create a custom (bespoke) autoscaling method` section for details.
 
+Using multiple metrics providers
+--------------------------------
+
+Paasta allows you to configure multiple metrics providers for your service, from the list above.  The service autoscaler
+will scale your service up if *any* of the configured metrics are exceeding their target value; conversely, it will
+scale down only when *all* of the configured metrics are below their target value.  You can configure multiple metrics
+providers using a list in the ``autoscaling.metrics_providers`` field, as follows:
+
+.. sourcecode:: yaml
+
+   ---
+   main:
+     cpus: 1
+     mem: 300
+     min_instances: 30
+     max_instances: 50
+     autoscaling:
+       metrics_providers:
+         - type: cpu
+           setpoint: 0.5
+         - type: active-requests
+           desired_active_requests_per_replica: 10
+
+There are a few restrictions on using multiple metrics for scaling your service, namely:
+
+1. You cannot specify the same metrics provider multiple times
+2. You cannot use bespoke autoscaling (see Decision Policies, above) with multiple metrics providers
```

Comment:
should we add a bullet point for cpu autoscaling + cpu autotune with multiple metrics?

Reply:
@nemacysts I wasn't sure if this was relevant given that this is ostensibly "open source" and other people "using" paasta wouldn't have paasta autotuner, but I guess a) the set of people using paasta that aren't yelp is approximately 0, and b), paasta validate will still fail even in this case.  :lol: